<?xml version="1.0" encoding="UTF-8"?>
<!-- 
   File name: mainWindow.ui
   Author: Hynek Šabacký (xsabac02)
   Date created: 18/11/2021
   Date last modified: 5/12/2021
-->
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCalendarWidget" name="calendarWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">  /* header row */
  QCalendarWidget QWidget { alternate-background-color:#444444; }
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled 
  {
  	font-size:20px;  
  	color: rgb(180, 180, 180);  
  	background-color: black;  
  	selection-background-color: rgb(64, 64, 64); 
  	selection-color: rgb(0, 255, 0); 
  }
   
  /* days in other months */
  /* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{ 
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); 
}

QCalendarWidget QAbstractItemView:disabled 
{ 
color: rgb(64, 64, 64); 
}

QCalendarWidget QWidget#qt_calendar_prevmonth
{
   qproperty-icon:url (&quot;./stylesheet-branch-closed.png&quot;);
}

QCalendarWidget QWidget#qt_calendar_nextmonth
{
   qproperty-icon:url (&quot;./stylesheet-branch-closed.png&quot;);
}</string>
           </property>
           <property name="firstDayOfWeek">
            <enum>Qt::Monday</enum>
           </property>
           <property name="gridVisible">
            <bool>true</bool>
           </property>
           <property name="horizontalHeaderFormat">
            <enum>QCalendarWidget::SingleLetterDayNames</enum>
           </property>
           <property name="verticalHeaderFormat">
            <enum>QCalendarWidget::NoVerticalHeader</enum>
           </property>
           <property name="navigationBarVisible">
            <bool>true</bool>
           </property>
           <property name="dateEditEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="today">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Today</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="dayLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:&quot;white&quot;;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); </string>
           </property>
           <property name="text">
            <string>Day</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="dayScrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:&quot;white&quot;;</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QTableView" name="dayTableView">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>390</width>
              <height>737</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="monthLabel">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:&quot;white&quot;;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); </string>
           </property>
           <property name="text">
            <string>Month</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="monthScrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:&quot;white&quot;;</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QTableView" name="monthTableView">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>389</width>
              <height>690</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="export">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Export as CSV</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
